tests/unittests/bases/test_ddp.py

```diff
@@ -48,47 +48,41 @@ def _test_ddp_cat(rank, worldsize):
     assert torch.all(torch.eq(dummy.foo, tensor([1, 1])))
 
 
-def _test_ddp_sum_cat(rank, worldsize):
-    setup_ddp(rank, worldsize)
+def _test_ddp_sum_cat(rank):
     dummy = DummyMetric()
     dummy._reductions = {"foo": torch.cat, "bar": torch.sum}
     dummy.foo = [tensor([1])]
     dummy.bar = tensor(1)
     dummy._sync_dist()
 
     assert torch.all(torch.eq(dummy.foo, tensor([1, 1])))
-    assert dummy.bar == worldsize
+    assert dummy.bar == NUM_PROCESSES
 
 
-def _test_ddp_gather_uneven_tensors(rank, worldsize):
-    setup_ddp(rank, worldsize)
+def _test_ddp_gather_uneven_tensors(rank):
```

Comment:
```suggestion
def _test_ddp_gather_uneven_tensors(rank: int, worldsize: int = NUM_PROCESSES):
```

Reply:
Thining about keeping it just with setting defaults...

Reply:
fine by me, added :)